Braking change introduced by minor version 0.3.13

https://docs.rs/socket2/0.3.12/socket2/struct.Domain.html
https://docs.rs/socket2/0.3.13/socket2/struct.Domain.html

functions missing
`socket2::Domain::ipv4()`
`socket2::Domain::ipv6()`

Getting all these building actix:
```
error[E0599]: no function or associated item named `ipv4` found for struct `socket2::Domain` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:491:43
    |
491 |         net::SocketAddr::V4(_) => Domain::ipv4(),
    |                                           ^^^^ function or associated item not found in `socket2::Domain`

error[E0599]: no function or associated item named `ipv6` found for struct `socket2::Domain` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:492:43
    |
492 |         net::SocketAddr::V6(_) => Domain::ipv6(),
    |                                           ^^^^ function or associated item not found in `socket2::Domain`

error[E0599]: no function or associated item named `stream` found for struct `socket2::Type` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:494:44
    |
494 |     let socket = Socket::new(domain, Type::stream(), Some(Protocol::tcp()))?;
    |                                            ^^^^^^ function or associated item not found in `socket2::Type`

error[E0599]: no function or associated item named `tcp` found for struct `socket2::Protocol` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:494:69
    |
494 |     let socket = Socket::new(domain, Type::stream(), Some(Protocol::tcp()))?;
    |                                                                     ^^^ function or associated item not found in `socket2::Protocol`

error: aborting due to 4 previous errors
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -137,6 +137,16 @@ impl Domain {
     /// Domain for IPv6 communication, corresponding to `AF_INET6`.
     pub const IPV6: Domain = Domain(sys::AF_INET6);
 
+    /// Domain for IPv4 communication, corresponding to `AF_INET`.
+    pub fn ipv4() -> Domain {
+        Domain::IPV4
+    }
+
+    /// Domain for IPv6 communication, corresponding to `AF_INET6`.
+    pub fn ipv6() -> Domain {
+        Domain::IPV6
+    }
+
     /// Returns the correct domain for `address`.
     pub fn for_address(address: SocketAddr) -> Domain {
         match address {

